src/components/patterns/hero-banner.tsx

```diff
@@ -67,7 +67,7 @@ const HeroBanner = ({
       {children && (
         <div
           className={twMerge(
-            "rs-px-5 rs-py-8 relative order-1 flex w-full flex-col bg-white lg:z-10 lg:w-[calc(50%_-_5rem)]",
```

Comment:
This is to fix this:
![image](https://github.com/user-attachments/assets/e1063d2e-b185-4efa-bd18-55c89baaaf58)

With the min height:
![image](https://github.com/user-attachments/assets/77eeab04-e9b7-4570-a707-34279d019ace)

Reply:
I thought that was the desired display.